admin/src/components/header/invite.js

```diff
@@ -173,6 +198,65 @@ const ChooseRegion = ({ value, onChange }) => {
   );
 };
 
+const ChooseCenter = ({ value, onChange, centers, onSelect }) => {
+  const { user } = useSelector((state) => state.Auth);
+
+  // useEffect(() => {
+  //   if (user.role === REFERENT_ROLES.HEAD_CENTER) {
+  //     return (
+  //       onChange({ target: { value: user.cohesionCenterId, name: "cohesionCenterId" } }),
+  //       onChange({ target: { value: user.cohesionCenterName, name: "cohesionCenterName" } })
+  //     )
+  //   }
+  //   return (
+  //     (e) => {
+  //       console.log('VALUE', JSON.stringify(e));
+  //       onChange({ target: { value: e._id, name: "cohesionCenterId" } });
+  //       onChange({ target: { value: e.value, name: "cohesionCenterName" } });
+  //       onSelect?.(e);
+  //     }
+  //   )
+  // }, []);
```

Comment:
on peut utiliser le `useEffect` pour mettre des valeurs par défaut si besoin.
Dans ce cas, si c'est un chef de centre on va preremplir avec son centre (et le `disabled` l'empeche de changer)
```suggestion
   useEffect(() => {
     if (user.role === REFERENT_ROLES.HEAD_CENTER) {
       return (
         onChange({ target: { value: user.cohesionCenterId, name: "cohesionCenterId" } }),
         onChange({ target: { value: user.cohesionCenterName, name: "cohesionCenterName" } })
       )
     }
   }, []);
```